src/app/shared/models/section.ts

```diff
@@ -15,7 +15,7 @@ export class Section {
     private section_name: string;
     private students: Student[];
     private instructor: string;
-    private quests: string[];
+    private quests: SectionQuest[];
```

Comment:
what is this section quest class? the quest mismo?

Reply:
Nope. SectionQuest is for quest inquiry only. It contains function like isParticipant(looks if a user is a participant of some quest), getQuestId... stuffs like that.